### PR TITLE
Feature/grid web

### PR DESCRIPTION
**Pull request by vagnersaraiva**

### CODE REVIEW

File: packages/docs/src/web/Grid/Grid.story.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+const maxBreakpointWidth = {
+  md: { width: '1179px' },
+  sm: { width: '959px' },
+  xs: { width: '599px' }
+};
+
+const mountGridItem = (selectBreakpoint: any, value: any, key: number) => {
+  return (
+    <GridItem key={key} item {...value}>
```
Comment (robertLichtnow) [inline packages/docs/src/web/Grid/Grid.story.tsx]: Remove the `item` prop. If it is required, then you should check on your props definition inside your component

File: packages/web/src/Components/GridContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React, { FunctionComponent } from 'react';
+
```
Comment (robertLichtnow) [inline packages/web/src/Components/GridContainer.tsx]: Remove this line

File: packages/web/src/Components/GridItem.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React, { FunctionComponent } from 'react';
+
```
Comment (robertLichtnow) [inline packages/web/src/Components/GridItem.tsx]: Remove this line

File: packages/web/src/Components/GridContainer.tsx (hunk 2 above)
Comment (vagnersaraiva) [reply packages/web/src/Components/GridContainer.tsx]: Remove? But GridContainder depends on them

File: packages/web/src/Components/GridItem.tsx (hunk 3 above)
Comment (vagnersaraiva) [reply packages/web/src/Components/GridItem.tsx]: Remove? But GridItem depends on them

**Pull request merged**